### PR TITLE
Update style of About Page for 5.6

### PR DESCRIPTION
**Pull request by ryelle**
Screenshots (click for larger):

| About | Credits | Freedoms | Privacy |
|-------|---------|-----------|--------|
| 16-about | 16-credits | 16-freedoms | 16-privacy |

<details>
<summary>Expand for a smaller screen About page screenshot</summary>

16-about-small

</details>

Trac ticket: https://core.trac.wordpress.org/ticket/51415

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -52,202 +64,150 @@
...
-				<p><?php _e( 'In WordPress 5.5, images wait to load until they’re just about to scroll into view. The technical term is ‘lazy loading’.' ); ?></p>
-				<p><?php _e( 'On mobile, lazy loading can also keep browsers from loading files meant for other devices. That can save your readers money on data — and help preserve battery life.' ); ?></p>
+		<div class="has-background-image" style="background-image: url('data:image/svg+xml,<?php echo rawurlencode( '<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 1035 884"><circle cx="503" cy="434" r="310" fill="#E3DAD1"/><circle cx="831" cy="204" r="204" fill="#D1CFE4"/><circle cx="113.5" cy="770.5" r="113.5" fill="#D1DEE4"/></svg>' ); ?>');">
+			<div class="about__section has-2-columns is-wider-left has-transparent-background-color">
+				<div class="column">
+					<h2><?php _e( 'Greater layout flexibility' ); ?></h2>
+					<p><?php _e( 'Bring your stories to life with more tools that let you edit your layout with or without code. Single column blocks, designs using mixed widths and columns, full-width headers, and videos in your cover block—make small change or big statements with equal ease!' ); ?></p>
```
Comment (sabernhardt) [inline src/wp-admin/about.php:78]: I think the plural "make small **changes**" was intended.

File: src/wp-admin/about.php [hunk 2]
```diff
@@ -52,202 +64,150 @@
...
+			</div>
+			<div class="column has-border" style="background-color:#e4d1d1;background-color:var(--global--color-red)">
+				<h3><?php _e( 'Accessibility statement template' ); ?></h3>
+				<p><?php _e( 'Even if you’re not an expert, you can start letting folks know about your site’s commitment to accessibility at the click of a button! The new <a href="%s">feature plugin</a> includes template copy for you to update and publish, and it’s written to support different contexts and jurisdictions.', '#' ); ?></p>
+			</div>
+			<div class="column has-border" style="background-color:#d1d1e4;background-color:var(--global--color-purple)">
+				<h3><?php _e( 'Built-in Patterns' ); ?></h3>
+				<p><?php _e( 'If you’ve not had the chance to play with block patterns yet, all default themes now feature a range of block patterns that let you master complex layouts with minimal effort. Customize the patterns to your liking with the copy, images and colors that fit your story or brand.' ); ?></p>
```
Comment (sabernhardt) [inline src/wp-admin/about.php:170]: lowercase: "Built-in patterns"

File: src/wp-admin/about.php [hunk 3]
```diff
@@ -52,202 +64,150 @@
...
+				<h3><?php _e( 'jQuery' ); ?></h3>
 				<p>
 					<?php
 					printf(
-						/* translators: %s: 'wp_get_environment_type' function name. */
-						__( 'WordPress now has a standardized way to define a site’s environment type (staging, production, etc). Retrieve that type with %s and execute only the appropriate code.' ),
-						'<code>wp_get_environment_type()</code>'
+						/* translators: %s: jQuery update test plugin link. */
+						__( 'Updates to jQuery in WordPress take place across three releases 5.5, 5.6 and 5.7. As we reach the mid-point of this process, run the <a href="%s">update test plugin</a> to check your sites for errors ahead of time.' ),
+						'https://wordpress.org/plugins/wp-jquery-update-test/'
```
Comment (sabernhardt) [inline src/wp-admin/about.php:195]: Please add a colon in "three releases: 5.5, 5.6 and 5.7."

**Comment by ryelle** [comment]
This was merged in [r49640](https://core.trac.wordpress.org/changeset/49640/) — closing the PR.

**Pull request closed without merging**